packages/storage/src/providers/s3/types/options.ts

```diff
@@ -224,6 +226,12 @@ export type UploadDataOptions = CommonOptions &
 		 * @default false
 		 */
 		preventOverwrite?: boolean;
+		/**
+		 * Indicates the algorithm used to create the checksum for the object.
+		 * This checksum can be used as a data integrity check to verify that the data received is the same data that was originally sent.
+		 * @default undefined
```

Comment:
I think the decription is a bit verbose since integrity check means `to verify that the data received is the same data that was originally sent`.
```suggestion
		/**
		 * Indicates the algorithm used to create the checksum for the object.
		 * The checksum created is for data integrity check by service-side.
		 * By default this checksum is not created.
		 * @default undefined
```